Chap_API_Data_Mgmt.tex

```diff
@@ -45,6 +45,20 @@ \section{Support Macros}
 
 \ac{PMIx} provides a set of convenience macros for creating, initiating, and releasing data buffers.
 
+%%%%
+\littleheader{Static initializer for the data buffer structure}
+\declaremacro{PMIX_DATA_BUFFER_STATIC_INIT}
+
+Provide a static initializer for the \refstruct{pmix_data_buffer_t} fields.
+
+\versionMarker{4.2}
+\cspecificstart
+\begin{codepar}
+PMIX_DATA_BUFFER_STATIC_INIT
```

Comment:
What arguments does this macro take? Should we make it a function per PR #383?

Reply:
Thanks. So similar to `PTHREAD_MUTEX_INITIALIZER` it is the right-hand side of the initialization for a variable declaration.
 * Ref: https://linux.die.net/man/3/pthread_mutex_init